common/views/themes/typography.ts

```diff
@@ -350,13 +350,23 @@ export const typography = css<GlobalStyleProps>`
 export function makeFontSizeClasses(): string {
   return breakpointNames
     .map(bp => {
-      return `@media (min-width: ${themeValues.sizes[bp]}px) {
-      ${Object.entries(fontSizesAtBreakpoints[bp])
-        .map(([key, value]) => {
-          return `.font-size-${key} {font-size: ${value}rem}`;
-        })
-        .join(' ')}
-    }`;
+      return `
+        @media (min-width: ${themeValues.sizes[bp]}px) {
+          ${Object.entries(fontSizesAtBreakpoints[bp])
+            .map(([key, value]) => {
+              return `.font-size-${key} {font-size: ${value}rem;}`;
+            })
+            .join(' ')}
+        }
+
+        @container card (min-width: ${themeValues.cardSizes[bp]}px) {
+          ${Object.entries(fontSizesAtBreakpoints[bp])
```

Comment:
I reused `fontSizesAtBreakpoints` here because they looked ok, but a more considered e.g. `cardFontSizesAtBreakpoints` might be better